### DISCUSSION TITLE
以下评估方式是怎么得出语法纠错最好是 Mengzi-T5-CSC的， Precision么

### DISCUSSION THREAD

**Discussion by smartmark-pro** [shibing624/pycorrector#506] [Q&A]
<img width="887" alt="image" src="https://github.com/user-attachments/assets/1238cb77-a5da-4eb9-9417-156ac6bdf366">

**Comment by shibing624** [comment]
F1

**Reply by smartmark-pro** [reply]
表里macbert F1最高啊
